capitalizeAndFilter.js

```diff
@@ -1,6 +1,10 @@
 //takes in an array of strings, capitalizes all the strings, and filters out any string that starts with F/f.
 
 const capitalizeAndFilter = (stringArray) => {
+  // for(let index = 0; index < stringArray.length; index++) {
```

Comment:
nice work so far, I think part of what I see commented out here is that it's not returning anything, and don't yet see the functionality to filter out a string starting with F/f.   So close though!